lms/djangoapps/course_home_api/outline/tests/test_view.py

```diff
@@ -437,3 +437,18 @@ def test_cannot_enroll_if_full(self):
         self.update_course_and_overview()
         CourseEnrollment.enroll(UserFactory(), self.course.id)  # grr, some rando took our spot!
         self.assert_can_enroll(False)
+
+    def test_optional_completion(self):
```

Comment:
Nit: can we use a more verbose name for this test, like "test_optional_completion_off_by_default"?

Reply:
good idea!